agr_indexer/src/main/java/org/alliancegenome/indexer/RestConfig.java

```diff
@@ -0,0 +1,41 @@
+package org.alliancegenome.indexer;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.alliancegenome.api.json.APIBeanSerializerModifier;
+import org.alliancegenome.core.config.ConfigHelper;
+import si.mazi.rescu.ClientConfig;
+import si.mazi.rescu.serialization.jackson.DefaultJacksonObjectMapperFactory;
+import si.mazi.rescu.serialization.jackson.JacksonObjectMapperFactory;
+
+import javax.ws.rs.HeaderParam;
+
+public class RestConfig {
```

Comment:
@cmpich you should be able to use the ObjectMapper from the RestDefaultObjectMapper in the curation.config class so that we don't need to maintain both.